plugins/arDominionB5Plugin/js/fullWidthTreeView.js

```diff
@@ -0,0 +1,320 @@
+          // Operations allowed:
+          // - Before and after drag and drop between siblings
+          // - Move core operations (node drop event)
+          return operation === 'create_node' ||
+            (operation === 'move_node'
+              && (more.core || (more.dnd
+              && node.parent === more.ref.parent
+              && more.pos !== 'i'))
+            );
+        }
+      }
+    };
+
+    // Declare listeners
+    // On ready: scroll to active node
+    var readyListener = function ()
+    {
+      var $activeNode = $('li[selected_on_load="true"]')[0];
+
+      pager.updateMoreLink($moreButton, $resetButton);
+
+      // Override default scrolling
+      if ($activeNode !== undefined)
+      {
+        $activeNode.scrollIntoView(true);
+        $('body')[0].scrollIntoView(true);
+      }
+    };
+
+    // On node selection: load the informationobject's page and insert the current page
+    var selectNodeListener = function (e, data)
+    {
+      // Open node if possible
+      data.instance.open_node(data.node);
+
+      // When an element is clicked in the tree ... fetch it up
+      // window.location = window.location.origin + data.node.a_attr.href
+      var url = data.node.a_attr.href;
+      $.get(url, function (response)
+      {
+        response = $(response);
+
+        // Insert new content into page
+        $('#main-column h1').first().replaceWith($(response.find('#main-column h1').first()));
+
+        // Add empty breadcrumb section if current page has none, but response does
+        if (!$('#main-column .breadcrumb').length && $(response.find('#main-column .breadcrumb').length))
+        {
+          var breadcrumbDestinationSelector = (treeViewCollapseEnabled)
+            ? '.full-treeview-section'
+            : '#fullwidth-treeview-row';
+
+          $(breadcrumbDestinationSelector).after($('<section>', {class: 'breadcrumb'}));
+        }
+        $('#main-column .breadcrumb').replaceWith($(response.find('#main-column .breadcrumb')));
+
+        // Replace description content
+        $('#main-column .row').replaceWith($(response.find('#main-column .row')));
+
+        // If translation links exist in the response page, create element, if necessary,
+        // and replace translation links in the current page with them
+        if (response.find('.translation-links').length && !$('.translation-links').length)
+        {
+          $('section.breadcrumb').after($('<div class="btn-group translation-links"></div>'));
+        }
+        $('.translation-links').replaceWith($(response.find('.translation-links')));
+
+        // Replace error message
+        $('#main-column > div.messages.error').remove();
+        $('#main-column .breadcrumb').after($(response.find('#main-column > div.messages.error')));
+
+        // Attach the Drupal Behaviour so blank.js does its thing
+
+        // TODO: commented this out - does it need replacing?
+        //Drupal.attachBehaviors(document)
```

Comment:
Agh, I faced a similar issue the other day ... https://github.com/artefactual/atom/blob/dev-test/issue-13530-b5-themes/plugins/arDominionB5Plugin/js/clipboard.js#L127-L136

Don't worry about it yet, the expander and other behaviors won't work for now after changing node, but I hope we can find a global solution at some point. For example, to hide the empty fields in the index page I hope we can live without that JS behavior, avoiding the empty field/heading display directly while rendering (@replaceafill), but we still need to share some things as you know.